lib/config/testdata/v23.xml

```diff
@@ -0,0 +1,16 @@
+<configuration version="22">
```

Comment:
Forgot to increment number. Shouldn't a test pick this up?

Reply:
It doesn't matter, the test does the migration, we just check that things after the migration are in the right places where we expect them to be.

Reply:
and specifically that test is pointless busywork when the config format hasn't changed. it was good the first few iterations and we changed things like how we represent devices etc.